### PR TITLE
ISSD-624 [C.P 2.0] Include the incident contacts in the email template

### PR DESCRIPTION
**Pull request by eduardo-diniz**
https://liferay.atlassian.net/browse/ISSD-624

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/HighPriorityContacts/SetupHighPriorityContact.jsx [hunk 1]
```diff
@@ -112,6 +114,10 @@ const SetupHighPriorityContact = ({
 			.catch(console.error);
 	}, [client, project, project.accountKey]);
 
+	if (setCurrentContact) {
+		setCurrentContact(currentHighPriorityContacts);
+	}
```
Comment (kevenleone) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/HighPriorityContacts/SetupHighPriorityContact.jsx:119]: Call `set` state during the render is really bad, can cause loops and unnecessary processing, this cannot me here

```suggestion
	if (setCurrentContact) {
		setCurrentContact(currentHighPriorityContacts);
	}
```

Comment (kevenleone) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/HighPriorityContacts/SetupHighPriorityContact.jsx:119]: You can use this `setCurrentContact` next to `setCurrentHighPriorityContacts`

Inside the useEffect

**Review by kevenleone — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/HighPriorityContacts/SetupHighPriorityContact.jsx (hunk 1 above)
Comment (eduardo-diniz) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/HighPriorityContacts/SetupHighPriorityContact.jsx:119]: thanks!

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward:force

**Comment by kevenleone** [comment]
ci:forward:force

**Pull request closed without merging**